server/paymentProviders/thegivingblock/index.js

```diff
@@ -96,7 +97,7 @@ export const processOrder = async order => {
   const cryptoToFiatFxRate = await getFxRate(order.data.customData.pledgeCurrency, order.currency);
   if (cryptoToFiatFxRate) {
     const totalAmount = Math.round(order.data.customData.pledgeAmount * cryptoToFiatFxRate);
-    await order.update({ totalAmount });
+    await order.update({ totalAmount, status: orderStatus.PENDING });
```

Comment:
Update to PENDING before or after or differently, as there is no guarantee to enter this condition at this stage.

Reply:
Ah, yes, I've changed this to update the order status before the `if` statement.